internal/client.go

```diff
@@ -58,13 +63,13 @@ func (client *Client) checkServers() {
 	}
 }
 
-func (client *Client) checkServer(host string) {
-	c := &http.Client{}
+func (client *Client) getServerData(host string) (*Msg, error) {
 	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/", host), nil)
 	if err != nil {
-		client.logTextbox.SetText(err.Error())
+		return nil, err
```

Comment:
This also fixes a subtle bug where is http.NewRequest failed, we would continue executing, resulting in more errors down below.  In practice that's not a real problem, but something to be aware of.